src/schema.rs

```diff
@@ -633,3 +633,39 @@ fn default_true() -> bool {
 fn default_false() -> bool {
     false
 }
+
+#[derive(serde_derive::Deserialize, Debug)]
+#[serde(deny_unknown_fields, rename_all = "kebab-case")]
+pub(crate) struct Repo {
+    pub org: String,
+    pub name: String,
+    pub description: String,
+    pub bots: Vec<String>, // TODO: should this be an enum?
+    pub access: RepoAccess,
+}
+
+impl Repo {
+    const VALID_ORGS: &'static [&'static str] = &["rust-lang"];
```

Comment:
```suggestion
    const VALID_ORGS: &[&str] = &["rust-lang"];
```

I think this should work?

Reply:
Unfortunately, not: https://github.com/rust-lang/rust/issues/51370

Reply:
I... am surprised I've never hit this before, but I guess that makes some sense.